lib/js/components/Badges/BadgeScore/BadgeScore.vue

```diff
@@ -20,98 +24,141 @@
 		'color': $color-fail-text,
-		'color-additional': $color-fail-text-weak,
+		'color-suffix': $color-fail-text-weak,
 	),
 	'warning': (
-		'border': $color-warning-border,
+		'background': $color-warning-background-medium,
+		'icon': $color-warning-icon,
 		'color': $color-warning-text,
-		'color-additional': $color-warning-text-weak,
+		'color-suffix': $color-warning-text-weak,
 	),
 	'inverted': (
-		'border': $color-inverted-border,
-		'color': $color-inverted-text,
-		'color-additional': $color-inverted-text,
+		'background': $color-default-background,
+		'icon': $color-neutral-icon-weak,
+		'color': $color-neutral-text-weak,
+		'color-suffix': $color-neutral-text-weak,
 	),
 	'neutral': (
-		'border': $color-neutral-border-strong,
+		'background': $color-neutral-background,
+		'icon': $color-neutral-icon-weak,
 		'color': $color-neutral-text-weak,
-		'color-additional': $color-neutral-text-weak,
+		'color-suffix': $color-neutral-text-weak,
+	),
+	'neutral-medium': (
+		'background': $color-neutral-background-medium,
+		'icon': $color-neutral-icon-strong,
+		'color': $color-neutral-text,
+		'color-suffix': $color-neutral-text,
 	),
 );
 
-@mixin setBadgeScoreColor($self, $border, $color, $color-additional) {
-	border-color: $border;
+@mixin setBadgeScoreColor($self, $background, $icon, $color, $color-suffix) {
+	background-color: $background;
 	color: $color;
-	#{$self}__additionalText {
-		color: $color-additional;
+	#{$self}__icon {
+		color: $icon;
+	}
+	#{$self}__suffix {
+		color: $color-suffix;
 	}
 }
 
 .badgeScore {
 	$self: &;
 
 	@include textBold();
-	@include headlineL();
+	@include headlineM();
 
 	@each $color-name, $color-map in $badge-score-colors {
 		&.-#{$color-name} {
 			@include setBadgeScoreColor(
 				$self,
-				map-get($color-map, 'border'),
+				map-get($color-map, 'background'),
+				map-get($color-map, 'icon'),
 				map-get($color-map, 'color'),
-				map-get($color-map, 'color-additional')
+				map-get($color-map, 'color-suffix')
 			);
 		}
 	}
 
 	border-radius: 4px;
-	border-width: 2px;
-	border-style: solid;
-	display: inline-block;
-	min-width: $badge-score-width;
-	padding: $space-xxxs $space-xxxxs;
-	text-align: center;
+	display: inline-flex;
+	flex: 1;
+	justify-content: center;
+	min-width: $badge-score-min-width;
+	padding: $space-xxxs $space-xxs;
+
+	&__text {
+		align-self: baseline;
+	}
+
+	&__icon {
+		margin-right: $space-xxxxxs;
+	}
+
+	&__suffix {
+		@include headlineS();
+
+		align-self: baseline;
+	}
 
 	&.-small {
 		@include headlineS();
 
-		min-width: $small-badge-score-width;
+		min-width: $badge-score-small-min-width;
+		#{$self}__suffix {
+			@include headlineXS();
+		}
 	}
 
 	&.-xsmall {
-		@include textXS();
+		@include textInfoM();
 
-		min-width: $x-small-badge-score-width;
-		border-width: 1px;
-		padding: $space-xxxxs;
+		min-width: $badge-score-x-small-min-width;
+		padding: $space-xxxs;
+
+		#{$self}__suffix {
+			@include textInfoM();
+		}
+	}
+
+	&.-full-width {
```

Comment:
we try to use camel case for class names

Reply:
Updated.